src/main/kotlin/dev/tmsoft/lib/params/sorting/Sorting.kt

```diff
@@ -0,0 +1,63 @@
+package dev.tmsoft.lib.params.sorting
+
+import dev.tmsoft.lib.params.Field
+import dev.tmsoft.lib.params.PathValues
+import dev.tmsoft.lib.params.SingleValue
+import org.jetbrains.exposed.sql.Column
+import org.jetbrains.exposed.sql.EnumerationNameColumnType
+import org.jetbrains.exposed.sql.Query
+import org.jetbrains.exposed.sql.SortOrder
+import org.jetbrains.exposed.sql.Table
+
+abstract class Sorting(val table: Table) {
+    private val fields: MutableList<Field> = mutableListOf()
+
+    fun fields(): List<Field> = fields
+
+    fun add(
+        name: String,
+        column: Column<*>? = null,
+        possibleValues: List<String>? = null
```

Comment:
What is this parameter for?

Reply:
for openApi -> swagger

Reply:
and what values are possible in this field? ASC, DESC?

Reply:
ASC,
DESC,
ASC_NULLS_FIRST,
DESC_NULLS_FIRST,
ASC_NULLS_LAST,
DESC_NULLS_LAST
@shustrik

Reply:
Ok, but it's a constant, I can't provide MY_SORT_ORDER